src/components/Avatar/Avatar.scss

```diff
@@ -0,0 +1,182 @@
+    }
+
+    40% {
+        transform: scale(1.25);
+    }
+
+    60% {
+        transform: scale(.9);
+    }
+
+    100% {
+        transform: scale(1);
+    }
+}
+
+$sizes: (
+    xs: (
+        avatar: ( size: 24px, font: 12px ),
+        status: ( size: 14px, font: 8px )
+    ),
+    sm: (
+        avatar: ( size: 36px, font: 16px ),
+        status: ( size: 20px, font: 12px )
+    ),
+    md: (
+        avatar: ( size: 48px, font: 22px ),
+        status: ( size: 26px, font: 18px )
+    ),
+    lg: (
+        avatar: ( size: 60px, font: 28px ),
+        status: ( size: 33px, font: 23px )
+    ),
+);
+
+$themes: (
+    light: $white,
+    dark: $charocal
+);
+
+.uir-avatar {
+    position: relative;
+    box-sizing: border-box;
+    display: inline-block;
+    font-family: $font;
+    text-align: center;
+    text-transform: uppercase;
+    vertical-align: middle;
+
+    @each $size, $details in $sizes {
+        $avatarDetails: map-get($details, "avatar");
+        $statusDetails: map-get($details, "status");
+
+        &.uir-avatar--#{$size} {
+            $avatarSize: map-get($avatarDetails, "size");
+            $avatarFont: map-get($avatarDetails, "font");
+            width: $avatarSize;
+            height: $avatarSize;
+            font-size: $avatarFont;
+            line-height: $avatarSize;
+
+            .uir-avatar-user-status {
+                $statusSize: map-get($statusDetails, "size");
+                $statusFont: map-get($statusDetails, "font");
+                width: $statusSize;
+                height: $statusSize;
+                font-size: $statusFont;
+                line-height: $statusSize;
+            }
+
+            &.uir-avatar--halo .uir-avatar-inner::before {
+                width: $avatarSize + 4px;
+                height: $avatarSize + 4px;
+            }
+        }
+    }
+
+    @each $theme, $background in $themes {
+
+        &.uir-avatar--#{$theme} {
+
+            .uir-avatar-user-status {
+                background: $background;
+            }
+        }
+    }
+
+    &:hover {
+        cursor: pointer;
+        opacity: .8;
+    }
+
+    &:focus {
+        outline: none;
+        opacity: .8;
+    }
+
+    &.uir-avatar--disabled {
+
+        &:hover,
+        &:focus {
+            cursor: default;
+            outline: none;
+            opacity: 1;
+        }
+    }
+
+    &.uir-avatar--halo {
+
+        &:hover {
+
+            .uir-avatar-inner::before {
+                animation-play-state: paused;
+            }
+        }
+
+        .uir-avatar-inner {
+            border-color: #fff;
+            border-style: solid;
+            border-width: 2px;
+
+            &::before {
+                position: absolute;
+                top: -2px;
+                left: -2px;
+                z-index: -1;
+                display: block;
+                content: "";
+                // background-image: radial-gradient(farthest-corner at 0 0, $pink 0%, $pink 75%, $yellow 80%);
```

Comment:
remove?

Reply:
thanks for flagging, I need to chat with Antoine about whether we need the halo bordered and if we do, do we want yellow (as per m-dash-ui) or pink to be more on brand